samcli/commands/deploy/guided_config.py

```diff
@@ -93,5 +100,10 @@ def _save_parameter_overrides(self, cmd_names, config_env, parameter_overrides,
             if _params:
                 samconfig.put(cmd_names, self.section, "parameter_overrides", " ".join(_params), env=config_env)
 
+    def _save_image_repositories(self, cmd_names, config_env, samconfig, image_repositories):
+        if image_repositories:
+            _image_repositories = [f"{key}={value}" for key, value in image_repositories.items()]
+            samconfig.put(cmd_names, self.section, "image_repositories", _image_repositories, env=config_env)
```

Comment:
But I need to save it in a certain format, thats why its this separate.

Reply:
AFAIR, if you just pass the arguments in `**kwargs` this will automatically add subsection and populate items under there. So we don't need special handling here.

The reason why we have this with the others, is because of the escaping of these arguments, which we won't be doing for this.